keras/src/optimizers/muon.py

```diff
@@ -88,7 +88,7 @@ def __init__(
         loss_scale_factor=None,
         gradient_accumulation_steps=None,
         name="muon",
-        exclude_layers=None,
+        exclude_layers=[],
```

Comment:
Initialize with None. Then do `exclude_layers = exclude_layers or []` in the constructor to turn None into empty list